django/apps/config.py

```diff
@@ -73,8 +73,7 @@ def _path_from_module(self, module):
         """Attempt to determine app's filesystem path from its module."""
         # See #21874 for extended discussion of the behavior of this method in
         # various cases.
-        # Convert paths to list because Python's _NamespacePath doesn't support
-        # indexing.
+        # Convert to list because __path__ may not support indexing.
```

Comment:
IMO this change is rather related with ticket-32355, because `_NamespacePath` supports indexing in Python 3.8+, see https://github.com/python/cpython/commit/ab9b31f94737895f0121f26ba3ad718ebbc24fe1. I added a separate PR #14192.

Reply:
Got it. I rebased on top of main, so this line isn't part of this PR anymore.